cristin-biobank/src/main/java/no/unit/nva/biobank/model/nva/BiobankMaterial.java

```diff
@@ -0,0 +1,33 @@
+package no.unit.nva.biobank.model.nva;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import no.unit.nva.biobank.model.cristin.CristinBiobankMaterial;
+
+public class BiobankMaterial {
+    public static final String MATERIAL_CODE = "materialCode";
+    public static final String OTHER_MATERIAL_DESC = "otherMaterialDescription";
+
+    @JsonProperty(MATERIAL_CODE)
+    private final String materialCode;
+    @JsonProperty(OTHER_MATERIAL_DESC)
+    private final String otherMaterialDescription;
+
+    public BiobankMaterial(String materialCode,
+                                  String otherMaterialDescription) {
+        this.materialCode = materialCode;
+        this.otherMaterialDescription = otherMaterialDescription;
+    }
+
+    public BiobankMaterial(CristinBiobankMaterial cristinMaterial) {
```

Comment:
While this is ok, I would rather have created a static method for creating an instance of BiobankMaterial from a different object:

`
public static Biobank fromCristinBiobank(CristinBiobank cristinBiobank) {...}
`

Or extract this to some mapping utils if there is no reason why the classes should know about each other.